controllers/backup_controller.go

```diff
@@ -294,55 +321,57 @@ func (r *BackupReconciler) deleteBackup(backup *veleroapi.Backup, ctx context.Co
 	}
 }
 
-// set all acm backup info
+// set all acm resources backup info
 func setBackupInfo(ctx context.Context, veleroBackup *veleroapi.Backup, c client.Client) {
 
 	backupLogger := log.FromContext(ctx)
-	var clusterResource bool = false
+	var clusterResource bool = true // we want cluster resources such as ManagedCluster and MCO observability
```

Comment:
@vbirsan I disagree here. We do not want to backup managedclusters and in general cluster scoped resources. Happy to talk about this in a call.

Reply:
I was planning to discuss this PR in the scrum today

Reply:
MCO observability is a cluster scoped resource and from Crystal's document I understand it needs to be backed up 
I am not trying to backup cluster resources in general, but the 2 mentioned here : ManagedCluster and MCO; they are the only ones included since the backup filter sets the CRD to backup; let's discuss today
https://docs.google.com/document/d/1NTVFZsHUzRbZ65LnN1ekh0vDcU86BT2gO8lvqoAmq0s/edit